apps/condo/domains/organization/schema/TokenAccessRight.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Generated by `createschema organization.TokenAccessRight 'name:Text; user:Relationship:User:CASCADE;'`
+ */
+
+const { Select } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { SERVICE_USER_FIELD } = require('@condo/domains/miniapp/schema/fields/accessRight')
+const access = require('@condo/domains/organization/access/TokenAccessRight')
+
+const { INTEGRATIONS_NAME, SBBOL_INTEGRATION_NAME } = require('../constants')
+
+
+const TokenAccessRight = new GQLListSchema('TokenAccessRight', {
```

Comment:
Is assum that you will create `GetTokenService` with `getToken` mutation.
Is it no so abstract?

Reply:
Not certainly in that way. We have access tokens to user data in external services that are stored with reference to the user. Therefore, the service will always require a userId to find the right token. The service was planned to be called GetAccessTokenByUserIdService